### ISSUE TITLE
Broken for Django 2.0.5

### ISSUE THREAD

**Issue by shnassar** [mozmeao/django-allow-cidr#6]
* Django Allow CIDR version: 0.3.0
* Django version: 2.0.5
* Python version: 3.7
* Operating System:

### Description

Trying to use django-allow-cidr in my project.

### What I Did
Configured django-allow-cidr with my project and got the following traceback:

```
Exception Value: `AllowCIDRMiddleware` object has no attribute `get_response`
```

Seems the following is missed  in `__init__`:
```
self.get_response = args[0]
```

**Comment by pmac** [comment]
So far I can't reproduce. I just merged PR #7 to have Travis test Django 2.0 in Python 3.7 and things are passing. Any further details you can provide on your setup?

**Comment by shnassar** [comment]
thanks for the quick responses!

i've been trying to repro it myself but can't seem to remember how exactly i ran into this.
 
i wasn't seeing any issues when testing locally, but as soon as i deployed things out to the cloud every single one of my requests were hitting https://github.com/mozmeao/django-allow-cidr/blob/master/allow_cidr/middleware.py#L47

i looked through the django docs (https://docs.djangoproject.com/en/2.0/topics/http/middleware/) and found them mentioning 
"Middleware factories must accept a `get_response` argument"

after adding the following everything started working as expected:
```
    def __init__(self, *args, **kwargs):
        self.get_response = args[0]
```

couple things to note, although i remember stepping through the middleware code and trying my cidr entries with `netaddr` directly but did not notice anything out of the oridnary:
- i'm using 2 domains for `ALLOWED_HOSTS`
- am using a `/16` `ALLOWED_CIDR_NETS`


also tiny nitpick but i originally came across your project here and the `ALLOW_CIDR_NETS` versus `ALLOWED_CIDR_NETS` typo had me confused for a few minutes xD


thanks again for the help!

**Comment by whyscream** [comment]
I'm having this same issue when using django 1.11.9, python 2.7

**Comment by pmac** [comment]
The fix for this has been released as `0.3.1` and is now on PyPI. Thanks all!
